lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,74 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
   margin: 0;
-  padding-top: 20px;
   position: relative; // to make a containing block for validation popovers
 }
 
-.nx-form--simple {
-  border-top: none;
-  padding: 0;
-}
+.nx-form-footer {
+  @include container-horizontal;
 
-// for a form that is the entirety of a modal
-.nx-form--modal-form {
-  padding-top: 0;
+  border-top: 1px solid #e9e9e9;
+  padding: $nx-spacing-xl $nx-spacing-l;
+  text-align: right;
 }
 
 .nx-form-group {
-  margin-bottom: 20px;
+  margin-bottom: 6px;
 }
 
 .nx-form-row {
   @include container-horizontal;
-  margin-bottom: 24px;
+  margin-bottom: 6px;
```

Comment:
See note above about this 6px value.

Reply:
> See note above about this 6px value.

https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/765e91c924d0f35c452be938c152fabc6a306bf4